diode-server/ingester/component.go

```diff
@@ -16,7 +16,7 @@ import (
 	"google.golang.org/grpc/reflection"
 	"google.golang.org/protobuf/proto"
 
-	"github.com/netboxlabs/diode/diode-server/gen/diode/v1/diodepb"
+	"github.com/netboxlabs/diode-sdk-go/diode/v1/diodepb"
```

Comment:
So the idea is that there will be no internal gen for diode-proto. All the code will exist in go sdk, is that right?

Reply:
To be precise: there will be no internal gen for ingester proto, only reconciler proto being generated into `diode-server/reconciler/v1/reconcilerpb`. Considered generating everything into diode-sdk-go as it's one `diode.v1` proto package, but somehow having reconciler there doesn't fit well, as it's more internal service. Thoughts?

Reply:
I think we should do either:
- generate everything in go-sdk and use the dependency
- generate both internally and does not depended on SDK  (and sdk generates only ingester)

Reply:
Variant 2 it is, thanks!